Packs/AzureSentinel/Integrations/AzureSentinel/Pipfile

```diff
@@ -4,17 +4,35 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "==5.0.1"
-pytest-mock = "*"
-requests-mock = "*"
-pytest-asyncio = "*"
-cryptography = "*"
+flake8 = "==3.9.2"
```

Comment:
Why have we changed this file?

Reply:
The azure sentinel integration runs with the following docker image - ```demisto/crypto:1.0.0.24380```

So this change is simply to update the Pipfile to the same Pipfile used in the docker image - https://github.com/demisto/dockerfiles/tree/master/docker/crypto

So other developers can quickly create a virtual environment with the most updated python version and dependancies when updating this integration somewhere in the future